classes/StandardControlsRegistry.php

```diff
@@ -237,7 +237,8 @@ protected function registerPartialControl()
     protected function registerSectionControl()
     {
         $ignoreProperties = [
-            'dynamic',
+            'stretch',
+            'grow',
```

Comment:
Stretch and grow shouldn't be standard properties if they're only supported by three field types. Whitelist rather than blacklist please.

Reply:
I'm confused by your comment?

I have removed the dropdown menu and there is no longer three options. It has been changed and updated to having two sepratae checkboxes as you requested.

![ssss](https://user-images.githubusercontent.com/17784082/61736896-ff2af280-ad7e-11e9-8354-59a09ed1cb07.png)

The current system blacklisted the stretch item, I just added the same with grow.

Reply:
@ayumihamsaki while the current system did that I would prefer it was the other way around where the controls that used it had to register it in their properties configuration rather than the controls that don't having to ignore it.

Reply:
@LukeTowers p.s. that means I would need to update all the other checkboxes in Builder, because they are setup the same way.

I will get the main pr done first and re-look at this afterwards.